### ISSUE TITLE
Ensure_future

### ISSUE THREAD

**Issue by ghost** [kharidiron/StarryPy3k#54]
http://stackoverflow.com/questions/30854576/ensure-future-not-available-in-module-asyncio

I was need to correct it

**Comment by kharidiron** [comment]
As the documentation says (as well as the link you posted):

> **Requirements**
> 
> Python 3.4.4 or greater is required. Test are only conducted on Python versions 3.4 and 3.5.

This is an unfortunate consequence of trying to make use of new functionality that is not provided in older revisions. `ensure_future` was introduced in 3.4.4 and is the better way to handle queuing of tasks. While I could write the code to be backwards-compatible with 3.4.3 and before by using `async`, that also puts me in a terrible position of writing code for deprecated functions.

Long story short, make sure you use 3.4.4 or newer. =\

**Comment by roccosette** [comment]
But i use python 3.4... Why you close it ?

**Comment by kharidiron** [comment]
I'm more than happy to reopen this ticket, but I need you to verify: what (full) version number of python are you running? Telling me 3.4 isn't enough. I need to know 3.4.X.

**Comment by ghost** [comment]
It's OK. I'm on 3.4.2 but i can't take a more recent version. I use legit version on debian...

**Comment by rubellyte** [comment]
You need to use Python 3.4.4 or higher in order to run StarryPy.
